pkg/volume/csi/csi_plugin.go

```diff
@@ -228,7 +230,7 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	}
 
 	// Initializing the label management channels
-	nim = nodeinfomanager.NewNodeInfoManager(host.GetNodeName(), host, migratedPlugins)
+	nim = nodeinfomanager.NewNodeInfoManager(host.GetNodeName(), host, migratedPlugins, nodeLister)
```

Comment:
I think we should not initialize nodeinfomanager at all if the informer is nil.  Also `nim` appears to be unused?

Reply:
Can you let me know which informer being nil is of concern ?

The nim is global variable.

Reply:
Sorry my first comment was not correct. We still need to initialize node info manager even if the CSINodeInfo feature gate is off, because nim still needs to manage Node objects.

Instead of passing in nodeLister as a new variable, can we just get it from the KubeletVolumeHost inside?